src/components/cards/cards.jsx

```diff
@@ -0,0 +1,284 @@
+import PropTypes from 'prop-types';
+import React, {Fragment} from 'react';
+import {FormattedMessage} from 'react-intl';
+import Draggable from 'react-draggable';
+
+import styles from './card.css';
+
+import nextIcon from './icon--next.svg';
+import prevIcon from './icon--prev.svg';
+
+import helpIcon from './icon--help.svg';
+import closeIcon from '../close-button/icon--close.svg';
+
+const CardHeader = ({onCloseCards, onShowAll, totalSteps, step}) => (
+    <div className={styles.headerButtons}>
+        <div
+            className={styles.allButton}
+            onClick={onShowAll}
+        >
+            <img
+                className={styles.helpIcon}
+                src={helpIcon}
+            />
+            <FormattedMessage
+                defaultMessage="All How-Tos"
+                description="Title for button to return to how-to library"
+                id="gui.cards.all-how-tos"
+            />
+        </div>
+        {totalSteps > 1 ? (
+            <div className={styles.stepsList}>
+                {Array(totalSteps).fill(0)
```

Comment:
We have a similarly awkward thing going on in www for a year selection. I think it'd be worth it to start using the `lodash.range` util for this kind of stuff for readability.